packages/consent/consent-tools/package.json

```diff
@@ -13,7 +13,7 @@
     "!**/test-helpers/**"
   ],
   "scripts": {
-    ".": "yarn run -T turbo run --filter=@segment/analytics-consent-tools",
+    ".": "yarn run -T turbo run --filter=@segment/analytics-consent-tools...",
```

Comment:
Just curious, when is this script run?

Reply:
It's just a helper for local development. I think I wrote about in the docs under development.

Reply:
Also forgot: we do use this in CI to lazily build specific packages